docs/platforms/x86_64_linux/x86_64_linux.md

```diff
@@ -122,7 +122,7 @@ Note that you can visit [Swagger Editor](https://editor.swagger.io/) to graphica
   - Same network connected among the devices.
   - Same Authentication key in /var/edge-orchestration/user/orchestration_userID.txt
     - To let the Edge Orchestration devices communicate with each other, each devices should have same authentication key in:
-`/var/edge-orchestration/data/cert/edge-orchestration.key` (Any cert file can be authentication key)
+`/var/edge-orchestration/certs/edge-orchestration.key` (Any cert file can be authentication key)
```

Comment:
I think it should be modified like suggestion below.
```suggestion
`/var/edge-orchestration/user/orchestration_userID.txt` (Any cert file can be authentication key)
```

Reply:
I'm planning on deleting this as it's not being used yet. But this needs to be verified.